arches_for_science/media/js/views/components/workflows/sample-taking-workflow/sampling-info-step.js

```diff
@@ -85,7 +85,9 @@ define([
         });
         
         this.samplingDate.subscribe(function(val){
-            self.samplingName(arches.translations.samplingActivityOf.replace('{physicalThingName}', self.physicalThingNameValue) + ' ' + val);
+            const name = arches.translations.samplingActivityOf.replace('{physicalThingName}', self.physicalThingNameValue) + ' ' + val;
+            self.samplingName()[arches.activeLanguage].value = name
+            self.samplingName(self.samplingName());
```

Comment:
Do we have the same problem in the two places from instrument-info-step.js in [this commit](https://github.com/jacobtylerwalls/arches-for-science-prj/commit/696415b5f86c7dcd2afbdae0b025ceea03bc81e1)?

Reply:
Thanks - good catch

Reply:
It turns out that instrument-info-step only has the string - rather than the localized object in the counter-part functions, so we don't have to make this change.